versions.tf

```diff
@@ -1,10 +1,21 @@
 terraform {
-  required_version = ">= 0.12.0"
-
+  required_version = ">= 0.12"
```

Comment:
@jamengual this needs to be updated to `>= 0.12.26` due to the new required_providers format.